RecoTracker/LST/src/alpaka/ES_ModulesDev.cc

```diff
@@ -0,0 +1,3 @@
+#include "RecoTracker/LST/interface/LSTModulesDev.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/typelookup.h"
+TYPELOOKUP_ALPAKA_TEMPLATED_DATA_REG(SDL::modulesBuffer);
```

Comment:
Hi @makortel
Is there a way to avoid having an explicit host-specific RecoTracker/LST/src/ES_ModulesDev.cc ?
For some reason this `TEMPLATED_DATA_REG` leads to appropriate symbols in `libRecoTrackerLSTCudaAsync.so`, but nothing in the `libRecoTrackerLSTSerialSync.so`

This behavior doesn't change if I remove `RecoTracker/LST/src/ES_ModulesDev.cc`.
Can LTO be at play in some way? 
I tried to add `<flags CXXFLAGS="-fno-lto"/>`, but it didn't make the CPU ES `edm::typelookup::className` symbols show up in the CPU/serial so file.

Reply:
> Is there a way to avoid having an explicit host-specific RecoTracker/LST/src/ES_ModulesDev.cc ?

No, the behavior is intentional (at least for now). We want the non-Alpaka modules to be able to consume the host-side EventSetup data products (analogously to host-side Event data products). From there it follows that the symbols of the host-side data products must be in the non-Alpaka library of `RecoTracker/LST`, and the `.cc` file containing the `TYPELOOKUP_DATA_REG()` for the host-specific data product must reside there.